ts/src/pro/bitget.ts

```diff
@@ -156,11 +156,15 @@ export default class bitget extends bitgetRest {
             topics.push (args);
             messageHashes.push ('ticker:' + symbol);
         }
-        const tickers = await this.watchPublicMultiple (messageHashes, topics, params);
+        const ticker = await this.watchPublicMultiple (messageHashes, topics, params);
+        let result = undefined;
         if (this.newUpdates) {
-            return tickers;
+            result = {};
+            result[ticker['symbol']] = ticker;
+        } else {
+            result = this.tickers;
         }
-        return this.filterByArray (this.tickers, 'symbol', symbols);
+        return this.filterByArray (result, 'symbol', symbols);
```

Comment:
in this case we never need to filter the newUpdates because it is already using watchMultiple